kolibri/utils/server.py

```diff
@@ -402,6 +402,9 @@ def get_status():  # noqa: max-complexity=16
 
         if response.status_code != 200:
             # Probably a mis-configured kolibri
+            sys.stderr.write("---Debug information---\n")
+            sys.stderr.write(response.text)
+            sys.stderr.write("\n-----------------------\n")
```

Comment:
was this intentionally using `sys.stderr` rather than `logger.error`?

Reply:
It was intentional, to avoid adding this information to kolibri.log, so it's not mixed with the web server error messages.
I had thought this was to be reported to the person executing the `status` command only, but I have not clear if it would be good to have it in the log file too. What do you prefer?

Reply:
I don't have a strong opinion - just wanted to make sure it was intentional